src/transformers/models/bert/modeling_bert.py

```diff
@@ -746,6 +746,8 @@ class BertPreTrainedModel(PreTrainedModel):
 
     def _init_weights(self, module):
         """Initialize the weights"""
+        if getattr(module, "_is_hf_initialized", False):
```

Comment:
won't it be simpler to add a wrapper that will do the checking and not put the onus of checking on the specific model's `init_weights`? so that the models don't need to be changed.

Reply:
so something like:
```
def wrapper(module):
  if getattr(module, "_is_hf_initialized", False):
    return
  model._init_weights(module)
  module._is_hf_initialized = True

model.apply(wrapper)
```